src/components/NotificationFeed/MarkAsRead.tsx

```diff
@@ -33,6 +33,8 @@ export const MarkAsRead: React.FC<MarkAsReadProps> = ({ onClick }) => {
       className={`rnf-mark-all-as-read rnf-mark-all-as-read--${colorMode}`}
       disabled={unreadCount === 0}
       onClick={onClickHandler}
+      type="button"
+      aria-label={t("markAllAsRead")}
```

Comment:
If I understand correctly, screen readers can use the button text so I think the aria-label is redundant here.